packages/patternfly-4/react-catalog-view-extension/sass/patternfly-react-extensions/_patternfly-react-extensions.scss

```diff
@@ -0,0 +1,7 @@
+/**
+  Patternfly React Extensions Partials
+*/
+@import 'variables';
+@import 'catalog-item';
+@import 'catalog-tile';
+@import 'vertical-tabs';
```

Comment:
Why are we adding `VerticalTabs` to the `catalog-view`-extension package?

Reply:
It's a part of the original PF3 extension package and OpenShift uses it.

Reply:
but as this is not a `catalog` component, I wouldn't expect to find it in the catalog extension pacakge.

Reply:
@dgutride suggested the name for this; I can change it to whatever you guys agree on.